### PR TITLE
Address Pre-Publishing API issues

### PR DESCRIPTION
**Pull request by talhazi**
issues from webexteams://im?space=09f43af0-53de-11ee-bb75-67e6029b768a&message=b8159110-f1e5-11ee-9c5c-6f9426c4578a

### CODE REVIEW

**Review by weilueluo — APPROVED** [review]

File: overview/intro.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-# Cisco Defense Orchestrator Public API Documentation
+# Cisco Defense Orchestrator API Documentation
 
 >**Note**:
-The Cisco Defense Orchestrator Public API is currently in beta and is subject to change. If you adopt a beta API, it may not be fully compatible with future versions.
+The CDO API is currently in beta and is subject to change. If you adopt a beta API, it may not be fully compatible with future versions.
 
 
-Cisco Defense Orchestrator (CDO) exposes a rich REST API. Internal and external users can leverage the API to programmatically integrate with the product. This document describes the semantics of the REST API. The public API exposes a curated subset of features available across multiple CDO internal REST APIs. Backwards compatibility of the public API is guaranteed using a versioning system.
+Cisco Defense Orchestrator (CDO) exposes a rich REST API. Internal and external users can leverage the API to programmatically integrate with the product. This document describes the semantics of the REST API. Backwards compatibility of the API is guaranteed using a versioning system.
```
Comment (siddhuwarrier) [inline overview/intro.md:9]: Maybe we reword this to:
Cisco Defense Orchestrator (CDO) exposes a rich REST API. This document describes the semantics of the REST API. Backwards compatibility of the API is guaranteed using a versioning system.

The internal and external users bit is jut BS :)

**Review by siddhuwarrier — APPROVED** [review]

File: overview/intro.md (hunk 1 above)
Comment (hazital) [reply overview/intro.md:9]: agree, will remove that sentence
